NewRelic/Config.php

```diff
@@ -27,7 +27,7 @@ class Config
     private $customParameters;
     private $deploymentNames;
 
-    public function __construct(string $name, string $apiKey, string $licenseKey = null, bool $xmit = false, array $deploymentNames = [])
+    public function __construct(string $name = null, string $apiKey = null, string $licenseKey = null, bool $xmit = false, array $deploymentNames = [])
```

Comment:
Do we really allow `Config` to be created without any arguments? Isnt this change similar to #174?

Reply:
yes, this should be fixed by #174